docs/subnets/reference-cli-commands.md

```diff
@@ -0,0 +1,517 @@
+    --force                     If true, ignores if the subnet has been published in the past, and attempts a forced publish.
+-h, --help                      help for publish
+    --no-repo-path string       Do not let the tool manage file publishing, but have it only generate the files and put them in the location given by this flag.
+    --repo-url string           The URL of the repo where we are publishing
+    --subnet-file-path string   Path to the Subnet description file. If not given, a prompting sequence will be initiated.
+    --vm-file-path string       Path to the VM description file. If not given, a prompting sequence will be initiated.
+```
+
+### stats
+
+Show validator statistics for the given subnet
+
+**Usage:**
+
+`avalanche subnet stats [subnetName] [flags]`
+
+**Flags:**
+
+```
+    --fuji fuji         print stats on fuji (alias for `testnet`)
+-h, --help              help for stats
+    --mainnet mainnet   print stats on mainnet
+    --testnet testnet   print stats on testnet (alias for `fuji`)
+```
+
+## Network
+
+The network command suite provides a collection of tools for managing
+local subnet deployments.
+
+When a subnet is deployed locally, it runs on a local, multi-node
+Avalanche network. Deploying a subnet locally will start this network
+in the background. This command suite allows you to shutdown,
+restart, and clear that network.
+
+This network currently supports multiple, concurrently deployed
+subnets.
+
+### clean
+
+The network clean command shuts down your local, multi-node network. All
+the deployed subnets will shutdown and delete their state. The network
+may be started again by deploying a new subnet configuration.
+
+**Usage:**
+
+`avalanche network clean [flags]`
+
+**Flags:**
+
+```
+    --hard   Also clean downloaded avalanchego and plugin binaries
+-h, --help   help for clean
+```
+
+### start
+
+The network start command starts a local, multi-node Avalanche network
+on your machine.
+
+By default, the command loads the default snapshot. If --snapshot-name flag
+is provided, that snapshot will be used for starting the network if
+it can be found. The command may fail if the local network is already
+running.
+
+**Usage:**
+
+`avalanche network start [flags]`
+
+**Flags:**
+
+```
+    --avalanchego-version string   use this version of avalanchego (ex: v1.17.12) (default "latest")
+-h, --help                         help for start
+    --snapshot-name string         name of snapshot to use to start the network from (default "default-1654102509")
+```
+
+### status
+
+The network status command prints whether or not a local Avalanche
+network is running and some basic stats about the network.
+
+**Usage:**
+
+`avalanche network status [flags]`
+
+**Flags:**
+
+```
+-h, --help   help for status
+```
+
+### stop
+
+The network stop command shuts down your local, multi-node network.
+
+All deployed subnets will shutdown gracefully and save their
```

Comment:
* Capitalize subnets
* Surround `snapshot-name` with backticks and add `--`
* Surround `network start --snapshot-name snapshotName` with backticks
* Surround `network start` with backticks